### ISSUE TITLE
Adding support for the Pytrack

### ISSUE THREAD

**Issue by Cinezaster** [ttn-be/ttnmapper#1]
I was looking into the LoPy and [Pytrack](https://www.pycom.io/product/pytrack/) combination to make a ttnmapper. It also adds battery capability which would be ideal to wear it or put it on my bike while I'm on the road.
I don't have a LoPy and Pytrack yet, because I don't master Python that well YET.

**Comment by mainini** [comment]
I'm running the LoPy with the normal pycom expansion board which indeed adds battery capability and works well so far. Without looking into the details, adjusting the code for the Pytrack board should be straightforward, as long as the GNSS module on the board outputs standard NMEA-0183 position data (which it almost certainly does).

I currently don't plan to buy a Pytrack board but I would happily adjust the code if provided with one or also give some limited support on doing so.

**Comment by Cinezaster** [comment]
@mainini I already learned microPython with this project and have a working LoPy + Pytrack TTNMapper. 
Your code was a great base to start from. But since I moved a different direction with my project, I won't push a pull request. 

My flow
1. I'll wait for a gps/glonass/... fix when I have the LED is blinking green
2. When I want to send my position and test for connectivity I'll have a pushbutton to do so.
3. I'll wait again, for a next push on the button
